faucet/valve.py

```diff
@@ -828,6 +828,7 @@ def ports_add(self, port_nums, cold_start=False, log_msg='up'):
                 continue
 
             if port.lacp:
+                port.dyn_lacp_port_id = ((self.dp.dp_id % 1000) * 100 + port.number) % 65535
```

Comment:
this does not at all seem like a sufficient range to avoid collisions.

Reply:
I used dp_id since it was readily available. Another way could be to use the last few digits of an md5 hash of the dp name. The suggested use of a list of dps makes sense but I odnt think that level of detail is exposed at the valve level.